app/models/repository_organisation.rb

```diff
@@ -57,6 +55,19 @@ class RepositoryOrganisation < ApplicationRecord
            :download_org_from_host_by_login, :download_repos, :download_members,
            :check_status, to: :repository_owner
 
+  # TODO: can this be an association again if we made projects_dependencies a Repository association again?
```

Comment:
How difficult is this to do? Seems like a lot (all?) of the calls to `projects_dependencies` in this PR are N+1s in the number of repositories

Reply:
looks like I'm able to turn `projects_dependencies` into an association, so this should be possible. I'll give it a go